input/core/src/commonMain/kotlin/majestic/calendar/days/tools/DayDefaultColors.kt

```diff
@@ -0,0 +1,19 @@
+package majestic.calendar.days.tools
+
+import androidx.compose.ui.graphics.Color
+import majestic.editor.tools.StateColors
+
+data class DayDefaultColors(
```

Comment:
Lets call the class `DayColors`

So that when the caller is trying to access them, the code reads `DayColors.Default` instead of `DayDefaultColors.Default`